src/lib/routers/history.ts

```diff
@@ -180,6 +203,8 @@ class BrowserHistory<TRouteState> implements Router<TRouteState> {
    * Removes the event listener and cleans up the URL.
    */
   public dispose(): void {
+    this.isDisposed = true;
```

Comment:
Do we have a test the assets the following flow:

```
start InstantSearch
make some changes (url changes)
dispose
try to make changes (not possible)
start InstantSearch again
make changes (url changes)
```

I'm assuming the router gets recreated on the next start of InstantSearch, but not 100% sure. Do you know?

Reply:
I added a test called `dispose-start-test.ts` with that flow, it pass